HLTrigger/HLTcore/interface/HLTPrescaleProvider.h

```diff
@@ -103,19 +103,32 @@ class HLTPrescaleProvider {
 private:
   void checkL1GtUtils() const;
   void checkL1TGlobalUtil() const;
+
   template <typename T>
   T convertL1PS(double val) const {
+    static_assert(std::is_same_v<T, double>,
+                  "\n\n\tPlease use convertL1PS<double> or convertL1PS<FractionalPrescale>"
```

Comment:
The static assert only allows type 'T=double', while in the message text you also allow for type 'FractionalPrescale'. This should be made consistent.

Reply:
It can be done, but it would be redundant. That is because the `FractionalPrescale` case is specialised explicitly in [HLTPrescaleProvider.cc#L107](https://github.com/cms-sw/cmssw/blob/165b65b30d75fe7e6bbdbafac7c2929046f7ead7/HLTrigger/HLTcore/src/HLTPrescaleProvider.cc#L107), so if `FractionalPrescale` is used, there is no `static_assert` call.